### PR TITLE
[LIB-277] As a user I can use custom webhook / codebox response in iOS library

### PR DESCRIPTION
**Pull request by trapi**

### CODE REVIEW

File: syncano-ios/SCCodeBox.m [hunk 1]
```diff
@@ -21,23 +21,21 @@ + (void)runCodeBoxWithId:(NSNumber *)codeBoxId params:(NSDictionary *)params onS
...
                 completion(nil,error);
+                return;
             }
-        } else {
-            if (completion) {
-                SCTrace *trace = [[SCTrace alloc] initWithJSONObject:responseObject andCodeboxIdentifier:codeBoxId];
-                completion(trace,error);
-            }
+            SCTrace *trace = [[SCTrace alloc] initWithJSONObject:responseObject andCodeboxIdentifier:codeBoxId];
+            completion(trace,error);
```
Comment (lifcio) [inline syncano-ios/SCCodeBox.m]: Good change! :+1:

File: syncano-ios/SCWebhook.m [hunk 2]
```diff
@@ -28,43 +28,95 @@ + (void)runWebhookWithName:(NSString *)name withPayload:(NSDictionary *)payload
...
++ (void)runPublicWebhookWithURLString:(NSString *)urlString params:(NSDictionary *)params completion:(SCWebhookCompletionBlock)completion {
+    SCAPIClient *apiClient = [[SCAPIClient alloc] initWithBaseURL:[NSURL URLWithString:urlString]];
+    
+    [self runPublicWebhookWithPath:@"" params:params usingAPIClient:apiClient completion:completion];
+}
+
+#pragma mark - Custom webhooks
++ (void)runCustomWebhookWithName:(NSString *)name completion:(SCCustomResponseCompletionBlock)completion {
+    SCAPIClient *apiClient = [[Syncano sharedAPIClient] copy];
+    apiClient.responseSerializer = [AFHTTPResponseSerializer serializer];
```
Comment (lifcio) [inline syncano-ios/SCWebhook.m]: To be sure - `[AFHTTPResponseSerializer serializer]` - based on comments on method declaration, this makes returned object be an instance of `NSData` ?

Comment (trapi) [inline syncano-ios/SCWebhook.m]: Exactly.
However I put "id" in SCCustomResponseCompletionBlock for those who want to use their custom serializer.

**Comment by lifcio** [comment]
Thanks, LGTM from me. Though, let's wait with merging for @jaskooo

**Comment by jaskooo** [comment]
@trapi Don't wait. Megre :) LGTM

**Pull request merged**